source/includes/telemetry-data.rst

```diff
@@ -0,0 +1,64 @@
+
+.. 
+   This file describes all telemetry data collected
+   by the {+product+}. Field names are not included
+   as they are not exposed through the public API.
+
+Your Code
+~~~~~~~~~
+
+The following is the information the {+product+} collects about your code:
+
+- The number of unique data types you use as generic type parameters in your
```

Comment:
I did not mention this in the doc: nested types and enums are counted as well. Maybe just "The number of unique data types used in your LINQ and builder expressions" would be more accurate?

Reply:
Discussed this with Boris on Slack, and we arrived at the phrasing "The number of user defined data types in your provided [mapping](https://mongodb.github.io/mongo-csharp-driver/2.14/reference/bson/mapping/)"